### PR TITLE
Add "addon" console command to enable and disable addons

### PR DESCRIPTION
**Pull request by mexon**
I'm loving the console command.  I use ansible to build a test server, and I'd like to use these console commands to do more detailed setup.  Is there any appetite for adding this "addon" command for enabling and disabling addons?

### CODE REVIEW

File: src/Console/Addon.php [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	 *
+	 * @return int Return code of this command
+	 *
+	 * @throws \Exception
+	 */
+	private function list()
+	{
+		$subCmd = $this->getArgument(1);
+		$start = $this->getOption(['s', 'start'], 0);
+		$count = $this->getOption(['c', 'count'], Pager::ITEMS_PER_PAGE);
```
Comment (MrPetovan) [inline src/Console/Addon.php]: These parameters aren't used and should be removed from here and the list command synopses.

File: src/Console/Addon.php [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+	private $dba;
+
+	protected function getHelp()
+	{
+		$help = <<<HELP
+console user - Modify addon settings per console commands.
+Usage
+	bin/console addon list all [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
+	bin/console addon list enabled [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
+	bin/console addon list disabled [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
```
Comment (MrPetovan) [inline src/Console/Addon.php]: Please remove the unused parameters start and count from these three command synopses.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by mexon** [comment]
Oh yeah, oops.  I was planning to implement that feature, but then didn't 😛  Maybe in a later revision.  Removed the flags.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
The addon list isn't bound to expand much more than what it is now, I don't think pagination is relevant for it.

Please note that I changed the target branch for this PR so that it will be included in the next release (2021.06) since we are currently in the Release Candidate period for the upcoming 2021.03 release when we fix bugs only.

**Comment by mexon** [comment]
Yep, agreed about the length of the addon list.  Putting it in the next release sounds good.  Thanks!

**Comment by MrPetovan** [comment]
Thank you for your work!

**Pull request merged**